### DISCUSSION TITLE
使用docker-compose启动milvus服务过程中，etcd容器启动报错

### DISCUSSION THREAD

**Discussion by javanlu123** [milvus-io/milvus#16790] [Q&A and General discussion]
操作系统：windows server 2019
docker版本：V20.10.9
docker-compose版本：V2.4.1
CPU/memory：16G
Milvus版本：2.0.1
启动docker-compose.yml文件过程中，etcd容器启动报错，导出其日志如下：
`{"level":"info","ts":1651114073.0140486,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_AUTO_COMPACTION_MODE","variable-value":"revision"}
{"level":"info","ts":1651114073.015824,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_AUTO_COMPACTION_RETENTION","variable-value":"1000"}
{"level":"info","ts":1651114073.0159514,"caller":"flags/flag.go:113","msg":"recognized and used environment variable","variable-name":"ETCD_QUOTA_BACKEND_BYTES","variable-value":"4294967296"}
{"level":"info","ts":"2022-04-28T02:47:53.016Z","caller":"etcdmain/etcd.go:72","msg":"Running: ","args":["etcd","-advertise-client-urls=http://127.0.0.1:2379","-listen-client-urls","http://0.0.0.0:2379","--data-dir","/etcd"]}
{"level":"info","ts":"2022-04-28T02:47:53.023Z","caller":"etcdmain/etcd.go:115","msg":"server has been already initialized","data-dir":"/etcd","dir-type":"member"}
{"level":"info","ts":"2022-04-28T02:47:53.023Z","caller":"embed/etcd.go:131","msg":"configuring peer listeners","listen-peer-urls":["http://localhost:2380"]}
{"level":"info","ts":"2022-04-28T02:48:03.039Z","caller":"embed/etcd.go:367","msg":"closing etcd server","name":"default","data-dir":"/etcd","advertise-peer-urls":["http://localhost:2380"],"advertise-client-urls":["http://127.0.0.1:2379"]}
{"level":"info","ts":"2022-04-28T02:48:03.039Z","caller":"embed/etcd.go:369","msg":"closed etcd server","name":"default","data-dir":"/etcd","advertise-peer-urls":["http://localhost:2380"],"advertise-client-urls":["http://127.0.0.1:2379"]}
{"level":"fatal","ts":"2022-04-28T02:48:03.039Z","caller":"etcdmain/etcd.go:203","msg":"discovery failed","error":"listen tcp: lookup localhost on 168.1.7.11:53: read udp 172.19.47.215:45601->168.1.7.11:53: i/o timeout","stacktrace":"go.etcd.io/etcd/server/v3/etcdmain.startEtcdOrProxyV2\n\t/tmp/etcd-release-3.5.0/etcd/release/etcd/server/etcdmain/etcd.go:203\ngo.etcd.io/etcd/server/v3/etcdmain.Main\n\t/tmp/etcd-release-3.5.0/etcd/release/etcd/server/etcdmain/main.go:40\nmain.main\n\t/tmp/etcd-release-3.5.0/etcd/release/etcd/server/main.go:32\nruntime.main\n\t/home/remote/sbatsche/.gvm/gos/go1.16.3/src/runtime/proc.go:225"}
`
该如何解决？

**Comment by LoveEachDay** [comment]
Could you follow the steps below:

1. docker-compose down
2. delete the `volumes` directory
3. docker-compose up -d

**Reply by javanlu123** [reply]
Thanks，I have tried this method but it did not help.I can deploy etcd service in win 10 and redhat os platform，but still failed in windows server 2019 which is use windows container.(I also hava  updated the etcd version)
